active/fundraising.md

```diff
@@ -2,31 +2,60 @@
 
 ## Scope of responsibilities
 
-Coordinates fundraising efforts, particularly around corporate and major donations.
+This working group coordinates fundraising efforts, particularly around corporate and major donations.
+
+The goals of the group are:
+
+- Acquire funding to secure the financial future of Django and support future growth of Django
+- Build and strengthen relationships between Django and donor sources
+- Raise awareness of the Django Web Framework and its community with the broader world
```

Comment:
With this one, I’m struggling to see why this is a responsibility of the fundraising group? Raising awareness feels like something some of the other groups could be doing.

Reply:
I think a natural byproduct of fundraising is raising awareness. It's not the primary goal, but it should be one of them. A sponsor who gives $10,000 without saying anything about Django is not the same as a sponsor who gives $10,000 and tells their communities about the donation and Django. There's a broader picture as to what fundraising could do. Making it clear to the working group that their efforts mean more than a the bottom line could help influence hard decisions.

Reply:
Visibility also attracts more funding, so I also see value in this.

Reply:
Sounds fair!